WinUIGallery/Samples/Data/ControlInfoData.json

```diff
@@ -701,6 +644,64 @@
             "GridView",
             "ListView"
           ]
+        },
+        {
+          "UniqueId": "StandardUICommand",
```

Comment:
Should this be before Swipecard or is the grouping of the UI commands intentional here?

Reply:
Figured `StandUICommand` and `XAMLUICommand` are related, so putting them closer together?

Reply:
Looking at these.. should those 2 pages be combined and called `UICommand`?